llarp/dht/messages/findintro.cpp

```diff
@@ -43,6 +49,10 @@ namespace llarp
       if(!bencode_start_dict(buf))
         return false;
 
+      // relay order
+      if(!BEncodeWriteDictInt("O", relayOrder, buf))
```

Comment:
encoding needs to be done in lexicographical order (with the current way things are done)